t3f/tensor_train_batch.py

```diff
@@ -126,24 +127,34 @@ def right_tt_rank_dim(self):
 
   def __str__(self):
     """A string describing the TensorTrainBatch, its TT-rank and shape."""
+
+
+
```

Comment:
Too many blank lines (AFAIK there should be none here, but maybe one is also OK)